config/webpack/webpack.desktop.ts

```diff
@@ -1,29 +1,28 @@
-const path = require('path');
-const webpack = require('webpack');
-const _ = require('underscore');
-
-const desktopDependencies = require('../../desktop/package.json').dependencies;
-const getCommonConfig = require('./webpack.common');
+import path from 'path';
+import type {Configuration} from 'webpack';
+import webpack from 'webpack';
+// eslint-disable-next-line @dword-design/import-alias/prefer-alias, import/no-relative-packages -- alias imports don't work for webpack
+import {dependencies as desktopDependencies} from '../../desktop/package.json';
+import type Environment from './types';
+import getCommonConfig from './webpack.common';
```

Comment:
Mentioning this again since you 👍 'd even though I said it's NAB. Are we updating all the `config`s to `configuration`s or not? Just curious
<img width="822" alt="image" src="https://github.com/Expensify/App/assets/165133/73e0ced0-af4b-4815-a6ac-77b360afa6e6">

Reply:
I'm going to change it, thanks